lib/commands.js

```diff
@@ -75,7 +63,9 @@ function list () {
     paths.map(fsReadDir)
   )
     .then((results) => {
-      return filterExe(filterBlinkm([].concat(...results)));
+      let mergedResults = [];
+      mergedResults = mergedResults.concat.apply(mergedResults, results);
```

Comment:
this is odd. `results` will always be an array, `mergedResults` will always be an empty array. It seems that `mergedResults` is unnecessary and `results` can be passed to the `filterBlinkm` function, or have i misunderstood ?